packages/manager/src/features/Kubernetes/KubeCheckoutBar/HACheckbox.tsx

```diff
@@ -31,6 +31,10 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   price: {
     marginTop: theme.spacing(),
+    '& h3': {
+      color: `${theme.font.normal} !important`,
+      fontFamily: '"LatoWebRegular", sans-serif',
```

Comment:
![Screen Shot 2022-03-28 at 4 57 32 PM](https://user-images.githubusercontent.com/6440455/160486704-54a9f446-061a-47af-b1fa-3347941ccabe.jpg)


I think it should be something like this
```suggestion
      color: `${theme.palette.text.primary} !important`,
      fontFamily: theme.font.normal,
```